Assignment_2/src/GeometryShapes/Circle.java

```diff
@@ -0,0 +1,44 @@
+package GeometryShapes;
+
+import java.util.Scanner;
+
+public class Circle implements Two_DimensionalShapes {
```

Comment:
It's a bad practice to include underscores in the class/interface names. They should simply follow Uppar CamelCase conventions